ui/app/AppLayouts/Chat/popups/community/CommunityDetailPopup.qml

```diff
@@ -15,19 +15,19 @@ StatusModal {
     id: root
 
     property var store
-    // Not Refactored Yet
-    property QtObject community: null // root.store.chatsModelInst.communities.observedCommunity
+    property QtObject community: root.store.communitiesModuleInst.observedCommunity
     property string communityId: community.id
     property string name: community.name
     property string description: community.description
     property int access: community.access
-    property string source: community.thumbnailImage
-    property int nbMembers: community.nbMembers
+    property string source: community.image || ""
+    // TODO members needs to be refactored on the backend
+    property int nbMembers: 0//community.nbMembers
     property bool ensOnly: community.ensOnly
     property bool canJoin: community.canJoin
     property bool canRequestAccess: community.canRequestAccess
     property bool isMember: community.isMember
-    property string communityColor: community.communityColor || Style.current.blue
+    property string communityColor: community.color || Style.current.blue
```

Comment:
@jrainville the same should be done here, remove this check. **But** since we don't have any "styling" instance on the backend that would help us to set appropriate "blue" based on the active theme in case a color is not defined (empty), then we should leave this check here for some better times. :)